src/main/perf/SearchPerfTest.java

```diff
@@ -262,6 +262,12 @@ private static void _main(String[] clArgs) throws Exception {
 
     final boolean verifyCheckSum = !args.getFlag("-skipVerifyChecksum");
     final boolean recacheFilterDeletes = args.getFlag("-recacheFilterDeletes");
+    final String vectorField;
+    if (args.hasArg("-vectorField")) {
+      vectorField = args.getString("-vectorField");
```

Comment:
Yeah +1 to just use a boolean flag instead and hardwire the `vector` field name here (too).